src/fastoad_cs25/models/handling_qualities/tail_sizing/compute_vt_area.py

```diff
@@ -37,6 +37,7 @@ def setup(self):
         self.add_input(
             "data:geometry:vertical_tail:MAC:at25percent:x:from_wingMAC25", val=np.nan, units="m"
         )
+        self.add_input("settings:geometry:vertical_tail:area_factor", val=1.0)
```

Comment:
```suggestion
        self.add_input("tuning:geometry:vertical_tail:area_factor", val=1.0)
```

Reply:
Corrected in [4b80055](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/49/commits/4b800553e8258931efb928e82aa882275b69fe91)